### PR TITLE
Temporal: removed $system* query options

### PR DESCRIPTION
**Pull request by ralfhandl**

### CODE REVIEW

**Review by HeikoTheissen — COMMENTED** [review]
How do these test cases work?

**Comment by ralfhandl** [comment]
On Windows follow the instructions in https://github.com/oasis-tcs/odata-abnf/tree/master/tools/PowerShell.

If you use VS Code, then use the "Check ABNF & watch" task: whenever you save a changed ABNF or test case file, the ABNF is automatically compiled and all tests are executed.

**Review by HeikoTheissen — APPROVED** [review]

File: abnf/odata-temporal-testcases.xml [hunk 1]
```diff
@@ -66,105 +68,35 @@
...
-    <Match>ID</Match>
-  </Constraint>
-  <Constraint Rule="primitiveNonKeyProperty">
-    <Match>Budget</Match>
-    <Match>Name</Match>
-    <Match>Title</Match>
-  </Constraint>
-  <Constraint Rule="streamProperty">
-  </Constraint>
+  <Constraint Rule="streamProperty" />
```
[the remark this answers is not in the document]
Comment (ralfhandl) [reply abnf/odata-temporal-testcases.xml:81]: Try removing it and see what happens 😁

Explanation: without this $expand=Department etc. would be interpreted as expanding a stream property by the parser.

File: abnf/odata-temporal-testcases.xml [hunk 2]
```diff
@@ -66,105 +68,38 @@
...
-  <TestCase Name="Temporal - from with date" Rule="odataRelativeUri">
-    <Input>Employees?$from=2012-07-26</Input>
+  <TestCase Name="Temporal - from and toInclusive with date" Rule="odataRelativeUri">
+    <Input>Employees?$from=2012-07-26&amp;$toInclusive=2012-07-26</Input>
   </TestCase>
-  <TestCase Name="Temporal - from with datetimestamp" Rule="odataRelativeUri">
-    <Input>Employees?$from=2012-07-26T09:00:00.00-08:00</Input>
+  <TestCase Name="Temporal - from and to with datetimestamp" Rule="odataRelativeUri">
+    <Input>Employees?$from=2012-07-26T09:00:00.00-08:00&amp;$to=2012-07-26T11:00:00.00-08:00</Input>
   </TestCase>
```
Comment (HeikoTheissen) [inline abnf/odata-temporal-testcases.xml:104]:
```suggestion
  <TestCase Name="Temporal - from and to with datetimestamp" Rule="odataRelativeUri">
    <Input>Employees?$from=2012-07-26T09:00:00.00-08:00&amp;$to=2012-07-26T11:00:00.00-08:00</Input>
  </TestCase>
  <TestCase Name="Temporal - from and toInclusive with datetimestamp" Rule="odataRelativeUri">
    <Input>Employees?$from=2012-07-26T09:00:00.00-08:00&amp;$toInclusive=2012-07-26T11:00:00.00-08:00</Input>
  </TestCase>
```

**Review by HeikoTheissen — CHANGES_REQUESTED** [review]
Test case for $toInclusive with timestamp

**Review by HeikoTheissen — APPROVED** [review]

**Pull request merged**